anime_downloader/sites/darkanime.py

```diff
@@ -0,0 +1,60 @@
+import json
+import re
+
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+
+class DarkAnime(Anime, sitename = 'darkanime'):
+    sitename = 'darkanime'
+
+    @classmethod
+    def search(cls, query):
+        soup = helpers.soupify(helpers.get('https://app.darkanime.stream/api/v1/animes', params={'term': query}).json()['animesHtml'])
+        soup = soup.find_all('a', href=True)
+        search_results = [
+            SearchResult(
+                title = x.find('h3').text.strip(),
+                url = 'https://app.darkanime.stream' + x['href'],
+                )
+            for x in soup
+            ]
+        return search_results
+    
+    def _scrape_episodes(self):
+        html = helpers.soupify(helpers.get(self.url).text)
+        eps = html.find('ul', class_='mt-4').find_all('li')
+        eps = ['https://app.darkanime.stream' + x.a['href'] for x in eps]
+        eps.reverse()
+        return eps
+    
+    def _scrape_metadata(self):
+            self.title = helpers.soupify(helpers.get(self.url).text).find_all('h2')[0].text.strip()
+            
+class DarkAnimeEpisode(AnimeEpisode, sitename='darkanime'):

```

Comment:
ill do a sort sources later

Reply:
woah, big brain time, i did find_all()[0] bc it has only 1 result, but i could just do a find()